pandoc.cabal

```diff
@@ -19,7 +19,7 @@ description:     Pandoc is a Haskell library for converting from one markup
                  .
                  - light markup formats (many variants of Markdown,
                    reStructuredText, AsciiDoc, Org-mode, Muse, Textile,
-                   txt2tags)
+                   txt2tags, Markua)
```

Comment:
I'd consider Markua a "variant of Markdown," so not needing separate mention here.

Reply:
We changed it accordingly